source/validator/contracts/Validator.sol

```diff
@@ -341,7 +342,7 @@ contract Validator {
     */
   function coreSwapChecks(
     Types.Order memory order
-    ) public view returns (uint256, bytes32[] memory ) {
+    ) internal view returns (uint256, bytes32[] memory ) {
```

Comment:
what was the reason for making this internal similar to coreDelegateChecks? my initial view was to keep all methods public/external and keep them callable. even if we're not using them, they may still be accessible.

Reply:
I didn't see them used anywhere else and limited scope.
I'll go ahead and swap it back since it makes sense to keep them accessible.